.github/workflows/issues-generation.yml

```diff
@@ -0,0 +1,24 @@
+name: Issues Generation
+
+concurrency: issues-generation
+
+on:
+  issue_comment:
+    types: [created]
+  pull_request_review_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened, reopened]
+
+jobs:
+  issue-generation-bot:
+    if: ${{ github.event.issue.pull_request || github.event.pull_request }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run Issue Generation
+        uses: ./.github/workflows/actions/issues-generation
+        id: issues-generation
+        with:
+          github-token: ${{ secrets.LEARNING_GITHUB_TOKEN }}
```

Comment:
I think this could probably use `GITHUB_TOKEN`? Not sure if that's matters much.

Reply:
This needs to be a token for @buildpack-bot since it's that account we are making all the operations as.